### ISSUE TITLE
Error when training code

### ISSUE THREAD

**Issue by ic085908** [jiejiang-jojo/edhc#2]
Could you give me a clue about TypeError: unsupported operand type(s) for -: 'float' and 'NoneType' on  utilities.py when running main_pytorch.py.

![image](https://user-images.githubusercontent.com/46443600/175837260-3b0dc6cb-9655-4773-ab4a-e1bd939b368e.png)

Thank you in advance.

**Comment by ic085908** [comment]
![image](https://user-images.githubusercontent.com/46443600/175837310-51056540-e37d-450c-a354-6886f24ce5b2.png)

**Comment by AliHaider20** [comment]
tensor value must be None.

**Comment by ic085908** [comment]
@AliHaider20  Thank you for your help. I would like to ask you how to adjust the tensor value to None. This function linked to outputs of main_pytorch.py.

![image](https://user-images.githubusercontent.com/46443600/175840763-e4c0f2e3-0eb3-4406-b2d3-cad4296d7cde.png)

**Comment by AliHaider20** [comment]
No, I guess I wrote it wrong. The tensor variable value shouldn't be None.
For Ex: None - 0.5 doesn't make sense because they are 2 different data types, that is why Python is raising an error. You need to debug why tensor variable has is giving None value.

**Comment by zhgqcn** [comment]
> No, I guess I wrote it wrong. The tensor variable value shouldn't be None. For Ex: None - 0.5 doesn't make sense because they are 2 different data types, that is why Python is raising an error. You need to debug why tensor variable has is giving None value.

Could you please tell me which paper is related to the code？
